The-SMIRNOFF-force-field-format.md

```diff
@@ -1,103 +1,288 @@
+See [this entry](https://en.wikipedia.org/wiki/List_of_XML_and_HTML_character_entity_references) for more details.
+
+## Units
+
+If not specified by a `*_unit` attribute, the [standard OpenMM unit system](http://docs.openmm.org/latest/userguide/theory.html#units) is assumed:
+* distances are specified in `nanometers`
+* mass is specified in `daltons`
+* time is specified in `picoseconds`
+* charge is specified in `elementary_charge`,
+* temperature is specified in `kelvin`
+* quantity is specified in `moles`
+* angles are specified in `radians`
+
+**QUESTION:** Should we *require* the units instead of making them optional in order to minimize the possibility for mistakes?
+
+Allowed values for units are given in [`simtk.unit`](https://github.com/pandegroup/openmm/blob/master/wrappers/python/simtk/unit/unit_definitions.py).
+
+## Functional forms
+
+The SMIRNOFF uses direct chemical perception to assign parameters for potential energy terms independently for each term.
+Rather than first applying atom types and then looking up combinations of atom types for each force term, the rules for directly applying parameters to atoms is compartmentalized in separate sections.
+The file consists of multiple `Force` sections, with each section specifying the typing rules used to assign parameters for that potential term:
+```XML
+<BondForce potential="harmonic" length_unit="angstroms" k_unit="kilocalories_per_mole/angstrom**2">
+   <Bond smirks="[#6X4:1]-[#6X4:2]" length="1.526" k="620.0"/>
+   <Bond smirks="[#6X4:1]-[#1:2]" length="1.090" k="680.0"/>
+   ...
+</BondForce>
+<AngleForce potential="harmonic" angle_unit="degrees" k_unit="kilocalories_per_mole/radian**2">
+   <Angle smirks="[a,A:1]-[#6X4:2]-[a,A:3]" angle="109.50" k="100.0"/>
+   <Angle smirks="[#1:1]-[#6X4:2]-[#1:3]" angle="109.50" k="70.0"/>
+   ...
+</AngleForce>
+...
+```
+Each `Force` section specifies the functional form for the interaction using the `potential` attribute.
+Common defaults are defined, but the goal is to eventually allow these to be overridden by alternative choices or even algebraic expressions in the future.
+The format will distinguish between functional forms available in all common molecular simulation packages and those experimental features available in a few packages (especially OpenMM, which supports a flexible set of custom forces defined by algebraic expressions) with an *EXPERIMENTAL* category.
+
+Many of the specific forces are implemented as discussed in the [OpenMM Documentation](http://docs.openmm.org/latest/userguide/theory.html); see especially [Section 19 on Standard Forces](http://docs.openmm.org/latest/userguide/theory.html#standard-forces) for mathematical descriptions of these functional forms.
+In some cases, typically for consistency with the AMBER force field philosophy motivating some of the authors, we do some manipulation of parameters from these files as discussed below in "Parameter sections".
```

Comment:
Does this mean manipulation between reading this file and applying the parameters to the molecules? I think on its own "manipulation of parameters" is unclear in this sentence.

Reply:
Maybe clearer would be "processing and manipulation of parameters loaded from these files before they are applied to molecules as discussed..."?

Reply:
LGTM.